src/main/resources/locales/en/media-account-rejection-confirmation.json

```diff
@@ -2,14 +2,22 @@
     "title": "Rejection",
```

Comment:
Event those these pages are only in English, they have Welsh versions of the json files too so should update them

Reply:
Done :)